kotlin/workflow-ui-core/src/main/java/com/squareup/workflow/ui/Named.kt

```diff
@@ -42,4 +42,16 @@ data class Named<W : Any>(
   override fun toString(): String {
     return "${super.toString()}: $key"
   }
+
+  companion object {
+    /**
+     * Calculates the [compatibility key][Named.key] for a given [value] and [name].
```

Comment:
Nit: the fact that this uses `Named` is significant documentation, I wouldn't hide it behind a differently-named link. But not a big deal.

Reply:
It's more that `Named` uses this method than the other way around. Inclined to leave it alone, I suspect I'll be back here for another round of tidying soon.